tests/sentry/api/serializers/test_team.py

```diff
@@ -67,41 +73,47 @@ def test_member_count_does_not_include_invite_requests(self):
         result = serialize(team, self.user)
         assert result["memberCount"] == 2
 
-    def test_member_access(self):
+    def test_member(self):
```

Comment:
🤔 

can we make this more descriptive of what we're testing on the team member?